fetching-water.md

```diff
@@ -0,0 +1 @@
+# Fetching water
```

Comment:
It was so good because I have nothing to change. Cool

Reply:
Are you sure?Thanks then.